bin/osg-notify

```diff
@@ -65,13 +66,13 @@ def generateFullMessage(subject, to, from_name, message, sign=True, keyid=None):
         msg.attach(basemsg)
         msg.attach(signmsg)
         msg['Subject'] = subject
-        msg['Bcc'] = ", ".join(to)
+        msg['Bcc'] = to
```

Comment:
Since we're now only sending to one person at a time, use the "To" field instead of the "Bcc" field for the recipient.

Reply:
@matyasselmeci, the current "To" is set to `"Open Science Grid Helpdesk" <help@opensciencegrid.org>` - do you think that should be kept also for To, or should the real recipient replace it?

Reply:
No, only use the real recipient.